### PR TITLE
feat: video source undo

### PR DESCRIPTION
**Pull request by tataihono**
# Description

### Issue

can't undo/redo changes on video source

### Solution

add command provider to video source and nested image library (for poster image)

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/Slider/Settings/Drawer/ImageEdit/ImageEdit.tsx [hunk 1]
```diff
@@ -128,18 +127,14 @@ export function ImageEdit({
     setOpen(false)
   }
 
-  async function createImageBlock(imageBlock: ImageBlock): Promise<void> {
+  async function createImageBlock(input: ImageBlockUpdateInput): Promise<void> {
```
Comment (Kneesal) [inline apps/journeys-admin/src/components/Editor/Slider/Settings/Drawer/ImageEdit/ImageEdit.tsx:130]: lint here

File: apps/journeys-admin/src/components/ThemeProvider/ThemeProvider.tsx [hunk 2]
```diff
@@ -15,10 +15,7 @@ export function ThemeProvider({
   nested
 }: ThemeProviderProps): ReactElement {
   return (
-    <MuiThemeProvider
-      theme={adminLight}
-      data-testid="JourneysAdminThemeProvider"
-    >
+    <MuiThemeProvider theme={adminLight}>
```
Comment (Kneesal) [inline apps/journeys-admin/src/components/ThemeProvider/ThemeProvider.tsx:18]: Is this change needed?

File: apps/journeys-admin/src/libs/useBlockRestoreMutation/index.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 export {
   useBlockRestoreMutation,
-  BLOCK_RESTORE
+  blockRestoreUpdate,
+  BLOCK_RESTORE_FIELDS
```
Comment (Kneesal) [inline apps/journeys-admin/src/libs/useBlockRestoreMutation/index.ts:4]: lint here

**Review by Kneesal — CHANGES_REQUESTED** [review]
lgtm, just lint

File: apps/journeys-admin/src/components/ThemeProvider/ThemeProvider.tsx (hunk 2 above)
Comment (tataihono) [reply apps/journeys-admin/src/components/ThemeProvider/ThemeProvider.tsx:18]: it gets rid of a bunch of console logs since data-testid is not a working attribute on theme provider.

**Review by Kneesal — APPROVED** [review]

**Review by Kneesal — APPROVED** [review]
